Jenkinsfile.download_logs

```diff
@@ -43,7 +43,6 @@ pipeline {
 
                 dir ('assisted-installer-deployment') {
                     sh "skipper run ./tools/create_triage_tickets.py --jira-access-token ${JIRA_ACCESS_TOKEN} -v"
-                    sh "skipper run ./tools/create_testgrid_tickets.py --jira-access-token ${JIRA_ACCESS_TOKEN} -v"
```

Comment:
What about `create_testgrid_tickets.py`? We still want it?

Reply:
I just noticed that you deleted it on https://github.com/openshift-assisted/assisted-installer-deployment/pull/179